### ISSUE TITLE
www.richdad.com - Unblocked layer

### ISSUE THREAD

**Issue by veronikasav** [AdguardTeam/AdguardFilters#8605]
http://www.richdad.com/Resources/Rich-Dad-Financial-Education-Blog/December-2017/When%E2%80%99s-the-Last-Time-You-Calculated-Your-Wealth-Nu.aspx?feed=RichDadNews

AdGuard does block the Ad but does not prevent the page from going dark (see screenshot).
But be aware that the Ad only appears when scrolling down to the lower third of the page.

Another issue is that switching on and off AdGuard several times shows an error message.

<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/27292111/34128310-ef9603ec-e448-11e7-8ebd-96c12d982667.png)

![image](https://user-images.githubusercontent.com/27292111/34128333-fee9fa56-e448-11e7-8138-152c877cf693.png)

</details><br/>

***System configuration***

Application version: 1.3.3.89
Device: iPhone
Platform: iOS
OS: 11.1.2

Filters subscriptions:
ID=0 Name="User filter" Version=1.0.0.0 Enabled=YES
ID=2 Name="English filter" Version=1.1.39.20 Enabled=YES
ID=3 Name="Spyware filter" Version=1.0.16.8 Enabled=YES
ID=4 Name="Social media filter" Version=1.0.26.32 Enabled=YES
ID=14 Name="Annoyances filter" Version=1.0.22.38 Enabled=YES
ID=10 Name="Filter unblocking search ads and self-promotion" Version=1.0.1.31 Enabled=YES
ID=6 Name="German filter" Version=1.0.38.67 Enabled=YES
ID=11 Name="Mobile Ads filter" Version=1.0.10.83 Enabled=YES
ID=12 Name="Safari filter" Version=1.0.3.84 Enabled=YES

Helpdesk ID:             1108721

[//]: # (This template is meant for missed ad/false positive reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)

**Comment by BlazDT** [comment]
Fixed overlay.

Other issue is mentioned here:

https://github.com/AdguardTeam/AdguardForiOS/issues/550
